README.md

```diff
@@ -75,6 +76,42 @@ methods:
 For definitions for the various fields, consult the corresponding template file for the type of data you are updating.
 
 
+### How to Register Downtime ###
+
+Downtime is a period of time for which one or more services you provide are unavailable.
+You should register downtime if one of these is true:
+
+-  your site is part of the WLCG
+-  your CE is one of the services affected
+
+1. Find the file that should contain downtime information about resources you own.
+   It is named `topology/<FACILITY>/<SITE>/<RESOURCE GROUP NAME>_downtime.yaml`.
+
+   For example, `topology/University of Wisconsin/GLOW/GLOW.yaml` has the corresponding downtime file
+   `topology/University of Wisconsin/GLOW/GLOW_downtime.yaml`.
+
+   If the downtime file does not exist, you will have to create it.
+
+   To find out what resource group a host is in, you can grep for the FQDN of the host with something like:
+
+   `grep -F "<FQDN>" topology/*/*/*.yaml | grep -Fv _downtime.yaml | grep -Fv SITE.yaml`
+
+   If the grep returns nothing, then you do not have that host registered in the topology data
```

Comment:
- "If the above command"
- "then the host is not registered in the OSG topology data"